src/Apps/Order2/Routes/Details/Components/WireTransferInfo.tsx

```diff
@@ -0,0 +1,132 @@
+import { Box, Spacer, Stack, Text } from "@artsy/palette"
+
+interface BankDetails {
+  accountNumber: string
+  iban?: string
+  routingNumber?: string
+  swift: string
+  sortCode?: string
+  bankAddress: {
+    name: string
+    street1: string
+    street2?: string
+    city: string
+    country: string
+  }
+  intermediaryBank?: {
+    name: string
+    swift: string
+  }
+}
+
+const BANK_DETAILS: Record<string, BankDetails> = {
+  GBP: {
+    accountNumber: "88005417",
+    iban: "GB30PNBP16567188005417",
+    swift: "PNBPGB2L",
+    sortCode: "16-56-7",
+    bankAddress: {
+      name: "Wells Fargo Bank, N.A. London Branch",
+      street1: "1 Plantation Place",
+      street2: "30 Fenchurch Street",
+      city: "London, United Kingdom, EC3M 3BD",
+      country: "United Kingdom",
+    },
+    intermediaryBank: {
+      name: "ING Brussels",
+      swift: "NWBKGB2LXXX",
+    },
+  },
+  EUR: {
+    accountNumber: "88005419",
+    iban: "GB73PNBP16567188005419",
+    swift: "PNBPGB2L",
+    bankAddress: {
+      name: "Wells Fargo Bank, N.A. London Branch",
+      street1: "1 Plantation Place",
+      street2: "30 Fenchurch Street",
+      city: "London, United Kingdom, EC3M 3BD",
+      country: "United Kingdom",
+    },
+    intermediaryBank: {
+      name: "ING Brussels",
+      swift: "BBRUBEBB010",
+    },
+  },
+  USD: {
+    accountNumber: "4243851425",
+    routingNumber: "121000248",
+    swift: "WFBIUS6S",
+    bankAddress: {
+      name: "Wells Fargo Bank, N.A.",
+      street1: "420 Montgomery Street",
+      city: "San Francisco, CA 94104",
```

Comment:
#non-blocking. We can make these an array of _lines_ and in the view simply print them all, to avoid getting into preciseness of the structure, like city here. Minor tho.